desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.tsx

```diff
@@ -135,20 +182,11 @@ const StorageBrowserTable = ({
             <span className="hue-storage-browser__table-cell-name">{record.name}</span>
```

Comment:
Not related to this change but we need to sanitize all the file and folder names, there's a deXSS function in utils/html/deXSS that you can use.

Reply:
Unless we use `dangerouslySetInnerHTML` I think react.js handles this for us.

Reply:
Indeed. Please ignore. For some reason I had mako in my mind.